examples/quickstart/jupyter-notebooks/README.md

```diff
@@ -65,8 +65,15 @@ Make sure you meet the following requirements before starting the Jupyter-based
     jupyter notebook --port 3001
     ```
 
-- An available Druid instance. You can use the `micro-quickstart` configuration
-  described in [Quickstart](https://druid.apache.org/docs/latest/tutorials/index.html).
+- The Python API client for Druid. Clone the Druid repo if you haven't already.
+Go to your Druid source repo and install `druidapi` with the following commands:
+
+  ```bash
+  cd examples/quickstart/jupyter-notebooks/druidapi
+  pip install .
```

Comment:
Not directly related to your change, but we have both `pip` and `pip3` referenced in this README file. It'd be good to just reference one version, perhaps `pip` as python2 is long deprecated now

Reply:
Good call, updated